pkg/v1/tkg/client/machine_deployment.go

```diff
@@ -55,18 +55,19 @@ type DeleteMachineDeploymentOptions struct {
 
 // NodePool a struct describing a node pool
 type NodePool struct {
-	Name             string                    `yaml:"name"`
-	Replicas         *int32                    `yaml:"replicas,omitempty"`
-	AZ               string                    `yaml:"az,omitempty"`
-	NodeMachineType  string                    `yaml:"nodeMachineType,omitempty"`
-	Labels           *map[string]string        `yaml:"labels,omitempty"`
-	VSphere          VSphereNodePool           `yaml:"vsphere,omitempty"`
-	Taints           *[]corev1.Taint           `yaml:"taints,omitempty"`
-	VMClass          string                    `yaml:"vmClass,omitempty"`
-	StorageClass     string                    `yaml:"storageClass,omitempty"`
-	Volumes          *[]tkgsv1alpha2.Volume    `yaml:"volumes,omitempty"`
-	TKR              tkgsv1alpha2.TKRReference `yaml:"tkr,omitempty"`
-	NodeDrainTimeout *metav1.Duration          `yaml:"nodeDrainTimeout,omitempty"`
+	Name                  string                    `yaml:"name"`
+	Replicas              *int32                    `yaml:"replicas,omitempty"`
+	AZ                    string                    `yaml:"az,omitempty"`
+	NodeMachineType       string                    `yaml:"nodeMachineType,omitempty"`
+	Labels                *map[string]string        `yaml:"labels,omitempty"`
+	VSphere               VSphereNodePool           `yaml:"vsphere,omitempty"`
+	Taints                *[]corev1.Taint           `yaml:"taints,omitempty"`
+	VMClass               string                    `yaml:"vmClass,omitempty"`
+	StorageClass          string                    `yaml:"storageClass,omitempty"`
+	Volumes               *[]tkgsv1alpha2.Volume    `yaml:"volumes,omitempty"`
+	TKR                   tkgsv1alpha2.TKRReference `yaml:"tkr,omitempty"`
+	NodeDrainTimeout      *metav1.Duration          `yaml:"nodeDrainTimeout,omitempty"`
+	MachineDeploymentBase string                    `yaml:"machineDeploymentBase,omitempty"`
```

Comment:
nit: Can this be renamed to `BaseMachineDeployment` or `BaseMachineDeploymentName` ?

Reply:
What about the command line flag?

Reply:
how about `base-machine-deployment` ?